backend/public/js/utils.js

```diff
@@ -96,3 +96,23 @@ function validateRecordDeletion(form) {
         });
     }
 }
+
+
+function validateUserCreation(form) {
+    $.ajax({
+        url: '/user/add',
+        type: 'POST',
+        contentType: 'application/x-www-form-urlencoded',
+        data: {
+            email: form.email.value,
+            username: form.username.value,
+            password: form.password.value
+        },
+        success:(response) => {
+            alert(response);
```

Comment:
Why are we giving the whole `response` object in the alert box?

Reply:
That's the output of this.
![image](https://user-images.githubusercontent.com/42294172/72526871-33dcba00-3889-11ea-876b-5acd6dfacf6a.png)

Here response is just a message.